xbmc/interfaces/legacy/File.h

```diff
@@ -199,7 +199,7 @@ namespace XBMCAddon
       ///
       seek(...);
 #else
-      inline long long seek(long long seekBytes, int iWhence) { DelayedCallGuard dg(languageHook); return file->Seek(seekBytes,iWhence); }
+      inline long long seek(long long seekBytes, int iWhence = 0) { DelayedCallGuard dg(languageHook); return file->Seek(seekBytes,iWhence); }
```

Comment:
Better use the more descriptive `SEEK_SET` define instead of `0`.
```suggestion
      inline long long seek(long long seekBytes, int iWhence = SEEK_SET) { DelayedCallGuard dg(languageHook); return file->Seek(seekBytes,iWhence); }
```

Reply:
updated as suggested